hooks/hookexecution/executor.go

```diff
@@ -0,0 +1,186 @@
+package hookexecution
+
+import (
+	"context"
+	"net/http"
+	"sync"
+
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/hooks"
+	"github.com/prebid/prebid-server/hooks/hookstage"
+)
+
+const (
+	EndpointAuction = "/openrtb2/auction"
+	EndpointAmp     = "/openrtb2/amp"
+)
+
+type entity string
```

Comment:
What is an entity?

Reply:
An entity specifies the type of object that was processed during the execution of the stage.
For most stages, the entity will be equal to one of the constant values. But for the `bidder-request` and `raw-bidder-response` stages, the entity is set to the bidder name. This is similar to what PBS-Java does.

Reply:
Thank you. I think a comment will make it clear.